service/gcs/bridge/bridge.go

```diff
@@ -268,15 +268,15 @@ func (b *bridge) shutdownContainer(message []byte) (*prot.MessageResponseBase, e
 	return response, nil
 }
 
-func (b *bridge) terminateProcess(message []byte) (*prot.MessageResponseBase, error) {
+func (b *bridge) signalProcess(message []byte) (*prot.MessageResponseBase, error) {
 	response := newResponseBase()
-	var request prot.ContainerTerminateProcess
+	var request prot.ContainerSignalProcess
 	if err := commonutils.UnmarshalJSONWithHresult(message, &request); err != nil {
 		return response, errors.Wrapf(err, "failed to unmarshal JSON for message \"%s\"", message)
 	}
 	response.ActivityID = request.ActivityID
 
-	if err := b.coreint.TerminateProcess(int(request.ProcessID)); err != nil {
+	if err := b.coreint.SignalProcess(int(request.ProcessID), request.Options); err != nil {
```

Comment:
We should handle the case where signal is missing since older hosts won't have the signal set. There's no reason for that host change to be a breaking change.